include/minisketch.h

```diff
@@ -186,19 +186,26 @@ class Minisketch
     std::unique_ptr<minisketch, Deleter> m_minisketch;
 
 public:
-    /** See minisketch_bits_supported(). */
+    /** Check whether the library supports fields of the given size. */
     static bool BitsSupported(uint32_t bits) noexcept { return minisketch_bits_supported(bits); }
 
-    /** minisketch_implementation_supported(). */
-    static bool ImplementationSupported(uint32_t bits, uint32_t implementation) noexcept { return minisketch_implementation_supported(bits, implementation); }
-
-    /** See minisketch_implementation_max(). */
+    /** Get the highest supported implementation number. */
     static uint32_t MaxImplementation() noexcept { return minisketch_implementation_max(); }
 
-    /** See minisketch_compute_capacity(). */
+    /** Check whether the library supports fields with a given size and implementation number.
+     *  If a particular field size `bits` is supported, implementation 0 is always supported for it.
+     *  Higher implementation numbers may or may not be available as well, up to MaxImplementation().
+     */
+    static bool ImplementationSupported(uint32_t bits, uint32_t implementation) noexcept { return minisketch_implementation_supported(bits, implementation); }
+
+    /** Given field size and a maximum number of decodable elements n, compute what capacity c to
+     *  use so that sketches with more elements than n have a chance no higher than 2^-fpbits of
+     *  being decoded incorrectly (and will instead fail when decoding for up to n elements).
+     *
+     *  See minisketch_compute_capacity for more details. */
     static size_t ComputeCapacity(uint32_t bits, size_t max_elements, uint32_t fpbits) noexcept { return minisketch_compute_capacity(bits, max_elements, fpbits); }
 
-    /** See minisketch_compute_max_elements(). */
+    /** Reverse operation of ComputeCapacity. See minisketch_compute_max_elements. */
     static size_t ComputeMaxElements(uint32_t bits, size_t capacity, uint32_t fpbits) noexcept { return minisketch_compute_max_elements(bits, capacity, fpbits); }
 
     /** Construct a clone of the specified sketch. */
```

Comment:
Maybe document that this function will abort if the provided minisketch is invalid. Same for the copy assignment below.

Reply:
Nice catch. I've instead added a commit that makes it work as expected when passing an invalid object.